noxfile.py

```diff
@@ -150,6 +150,16 @@ def lint(session):
     session.run("pre-commit", "run", *args)
 
 
+@nox.session
+def vendoring(session):
+    # Required, otherwise we interpret --no-binary :all: as
+    # "do not build wheels", which fails for PEP 517 requirements
+    session.install("-U", "pip>=19.3.1")
```

Comment:
Can we drop this? The next version of vendoring will explicitly declare this dependency, and for now, we can just depend on "we get the latest version" behavior. 🤷🏻‍♂️

Reply:
I'm happy to do so, I only added it to replicate tox. I'll update the PR and then merge. Thanks for the review :-)